### PR TITLE
Removed Mocha/Chai. Added Jest, Webpack. Changed index.test.js

### PR DESCRIPTION
**Pull request by shenbomo**
I did not include all the boilerplate(placeholder) code because I did not yet find the equivalence of `context(...)`.

### CODE REVIEW

**Comment by shenbomo** [comment]
2 more commits were added to fix indentation and to add boilerplate code.

**Comment by shenbomo** [comment]
CI is still failing on `npm test`, very unstable.

File: src/model/index.test.js [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
+    test('contains one operator, initially set to empty string', () => {
+      expect(this.instance.operator).toBe('');
     });
-  })
+  });
 
   describe('#receiveOperand', () => {
-    context('when receiving first input', () => {
+    describe('when receiving first input', () => {
       it('first operand will be assied with the input value', () => {
```
Comment (shogunsea) [inline src/model/index.test.js]: `it` unchanged?

**Comment by shogunsea** [comment]
:shipit:

**Pull request merged**